plugins/vendor/webex/infra/kWebexAPIClient.php

```diff
@@ -5,7 +5,9 @@
  */
 class kWebexAPIClient extends kVendorClient
 {
-	const DELETE_SUCCESSFUL_CODE = 204;
+	const DELETE_RECORDING_SUCCESSFUL_CODE = 204;
+	const GET_TRANSCRIPT_SUCCESSFUL_CODE = 200;
```

Comment:
i dont think we need two enums for status code 200, its just. a successful http code status